core/client/fs/src/main/java/alluxio/client/block/stream/NettyPacketWriter.java

```diff
@@ -220,12 +229,12 @@ public void cancel() {
   @Override
   public void flush() throws IOException {
     mChannel.flush();
-
+    long pos = 0;
```

Comment:
I don't think is variable is necessary anymore, since we already do the `== 0` check in the try block.

Reply:
Deleted the unnecessary check `if (pos == 0） return` later.
But this variable is used in writing the write request.

Changed to
```
    long pos;
    try (LockResource lr = new LockResource(mLock)) {
      if (mPosToQueue == 0) {
        return;
      }
      while (mPosToWrite != mPosToQueue) {
        ......
      }
      pos = mPosToQueue;
    } catch (InterruptedException e) {
      .....
    }
    if (mEOFSent || mCancelSent) {
      return;
    }
    Protocol.WriteRequest writeRequest =
        mPartialRequest.toBuilder().setOffset(pos).setFlush(true).build();
```

Thanks!